x-pack/plugins/ml/public/application/components/model_snapshots/model_snapshots_table.tsx

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+
+// @ts-ignore
+import { formatDate } from '@elastic/eui/lib/services/format';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiInMemoryTable,
+  EuiLoadingSpinner,
+  EuiOverlayMask,
+  EuiConfirmModal,
+  EuiBasicTableColumn,
+} from '@elastic/eui';
+
+import { checkPermission } from '../../capabilities/check_capabilities';
+import { EditModelSnapshotFlyout } from './edit_model_snapshot_flyout';
+import { RevertModelSnapshotFlyout } from './revert_model_snapshot_flyout';
+import { ml } from '../../services/ml_api_service';
+import { JOB_STATE, DATAFEED_STATE } from '../../../../common/constants/states';
+import {
+  ModelSnapshot,
+  CombinedJobWithStats,
+} from '../../../../common/types/anomaly_detection_jobs';
+
+const TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss';
```

Comment:
We have several places with the same constant for time format. Shall we create a common one?

Reply:
Changed in ac22c8ce86162fd9d4db4687850de4ae33873f4b